### PR TITLE
Issue 329

### PR DESCRIPTION
**Pull request by hsnice16**
## Related Issue

Right now, on the hovering nav icon, the cursor is changing to pointer but on clicking it, nothing happens. So I think on clicking it route should change to '/' (or home page).

And in the footer, on hovering social handlers only the cursor is changing to a pointer type, if we also transform its size then it will look nicer.

Closes: #329

### Describe the changes you've made

Converted Nav-icon into a **Next** Link, and added code for hovering state of social-handles.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Tested on localhost.

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly whereever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by hsnice16** [comment]
Please review and merge.

And don't know why showing 8 commits? I did only one.

File: src/components/Nav/Navbar.jsx [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
 const Navbar = () => {
   return (
     <div className={styles.stylenavbar}>
-      <div className={styles.logo}>
-      <img className= {styles['header-logo']} src="/logo/codetrophs.png" alt="" />
-      </div>
+      <Link href='/feed'>
+        <a>
+          <div className={styles.logo}>
+            <img className= {styles['header-logo']} src="/logo/codetrophs.png" alt="" />
```
Comment (Anurag5086) [inline src/components/Nav/Navbar.jsx:13]: Add alt value

**Review by Anurag5086 — CHANGES_REQUESTED** [review]
Make sure the build check passes.

File: src/components/Nav/Navbar.jsx (hunk 1 above)
Comment (hsnice16) [reply src/components/Nav/Navbar.jsx]: Thanks, I did this change.

**Comment by hsnice16** [comment]
Please, review once more.

**Comment by hsnice16** [comment]
Thanks

**Comment by hsnice16** [comment]
Will PR's, without labels count for GSSOC?

**Pull request merged**